index.html

```diff
@@ -100,5 +100,20 @@ <h3 class="content_area2">Why Choose Us</h3>
 
       </form>
     </section>
+
+    <section>
+      <footer class="footer">
+        <h4>Connect with us</h4>
+        <div>
```

Comment:
Consider adding class names, because there is a likelihood that you may add an image in the footer element in the future. This can break your style. Since you are using `.footer img`.